mayastor/src/replica.rs

```diff
@@ -369,6 +394,37 @@ impl Replica {
             .send(errno_result_from_i32(lvol_ptr, errno))
             .expect("Receiver is gone");
     }
+
+    /// Callback called from SPDK for snapshot create method.
+    extern "C" fn snapshot_done_cb(
+        nvmf_req_ptr: *mut c_void,
+        _lvol_ptr: *mut spdk_lvol,
+        errno: i32,
+    ) {
+        let rsp: &mut spdk_nvme_cpl = unsafe {
+            &mut *spdk_sys::spdk_nvmf_request_get_response(
+                nvmf_req_ptr as *mut spdk_nvmf_request,
+            )
+        };
+
+        unsafe {
+            rsp.__bindgen_anon_1.status.set_sct(0); // SPDK_NVME_SCT_GENERIC
```

Comment:
hmm is there are no methods available to read this fields etc, we should consider adding small C wrappers to this.